pymilvus/client/grpc_handler.py

```diff
@@ -738,7 +738,7 @@ def can_loop(t) -> bool:
             if progress >= 100:
                 return
             time.sleep(DefaultConfigs.WaitTimeDurationWhenLoad)
-        raise MilvusException(message="wait for loading partition timeout")
+        raise MilvusException(message="wait for loading partition timeout, collection " + collection_name + ", partitions" + partition_names)
```

Comment:
can be simplified to:
`f"wait for loading partition timeout, collection: {collection_name}, partitions: {partition_names}"`